docs/reference/dlm/tutorial-manage-existing-data-stream.asciidoc

```diff
@@ -0,0 +1,59 @@
+[role="xpack"]
+[[tutorial-manage-existing-data-stream]]
+=== Tutorial: Update existing data stream
+
+preview::[]
+
+To update the lifecycle of an existing data stream you do following actions:
+
+. <<set-lifecycle>>
+. <<delete-lifecycle>>
+
+[discrete]
+[[set-lifecycle]]
+==== Set a data stream's lifecycle
+
+To add or to change the retention period of your data stream you can use the <<dlm-put-lifecycle, PUT lifecycle API>>.
+
+* You can set infinite retention period, meaning that your data should never be deleted. For example:
++
+[source,console]
+----
+PUT _data_stream/my-data-stream/_lifecycle
+{ } <1>
+----
+// TEST[setup:my_data_stream]
+<1> An empty payload means that your data stream is still managed and able to support your streaming data but your
+data will never be deleted.
+
+* Or you can set the retention period of your choice. For example:
++
+[source,console]
+----
+PUT _data_stream/my-data-stream/_lifecycle
+{
+  "data_retention": "30d" <1>
+}
+----
+// TEST[continued]
+<1> The retention period of this data stream is set to 30 days. This means that {es} is allowed to delete data that is
+older than 30 days at its own discretion.
+
+[discrete]
+[[delete-lifecycle]]
+==== "Unmanage" a data stream
```

Comment:
Shall we call this something along the lines of "Remove lifecycle for a data stream" and mention that it becomes unmanaged in the description?

Reply:
Done